VTEX - Promotions & Taxes API.json

```diff
@@ -4955,7 +4955,7 @@
                     "Prices (Legacy - v1)"
                 ],
                 "summary": "Get Price by SKU ID",
-                "description": "Price by SKU ID              \n> If your account is using Pricing v2, you should avoid using these routes. Please refer directly to the [Pricing v2 API](https://documenter.getpostman.com/view/101975/vtex-pricing-api/6YsWxKT) \n\n> If you are still using Pricing v1, please [check if your store is able to migrate to take advantage of many more features](https://help.vtex.com/en/faq/how-to-migrate-a-store-to-pricing-v2)",
+                "description": "Price by SKU ID              \n> If your account is using Pricing v2, you should avoid using these routes. Please refer directly to the [Pricing v2 API](https://documenter.getpostman.com/view/101975/vtex-pricing-api/6YsWxKT)",
```

Comment:
Esse link dá 404. Por que não colocar o link da API ref da [Pricing API](https://developers.vtex.com/docs/api-reference/pricing-api), que já é a v2?

Outras considerações:
- Será que não dá para remover esses endpoints da doc por completo? Ainda tem alguém usando ou está totalmente deprecado?
- Se não for o caso de remover, faria sentido tirar da Promotions & Taxes API e criar um arquivo separado para isso?

Reply:
Atualizei o link da Pricing v2.

Sobre as considerações:

- Os endpoints ainda não foram deprecados e não há previsão para isso.
- Faz sentido retirar de Promotions & Taxes API e criar um arquivo exclusivo. Vou abrir outra branch para isso :)